test/elm/convert/test.coffee

```diff
@@ -391,3 +390,135 @@ describe 'ToBoolean', ->
 
   it "should return false for F", ->
     should(@upperCaseF.exec(@ctx)).be.false
+
+describe 'ConvertsToBoolean', ->
+  @beforeEach ->
+    setup @, data
+
+  it "should return true for y", ->
+    should(@isTrueWithTrueValue.exec(@ctx)).be.true
```

Comment:
These all need the `()` at the end to actually call the functions.  Also, it's better to avoid the should().be form except for null, when it's necessary (see screenshots below).  Right now these lines just return `undefined` so the tests pass when some of them should fail.

![image](https://user-images.githubusercontent.com/14879344/55361502-5beb6000-54a5-11e9-98a9-0bdeb162724a.png)
![image](https://user-images.githubusercontent.com/14879344/55361518-69084f00-54a5-11e9-860a-4f7a212d6f02.png)

Reply:
This is a great point! It's a bit worrisome that so many checks in this file were already missing the `()`. Also, is there another reason why we should avoid the `should().be`, or is it just so this mistake doesn't get made?

Reply:
I'm just lurking, but... one of the things I really like about `should` is that your tests can read like English.  So the reason I avoid `should(...).be` is that I find `myVar.should.be.true()` much more natural than `should(myVar).be.true()`.

That said, if I knew then what I know now, I would have just used chai (for which `true` is a property not a function).  I bet whoever wrote the test above usually writes tests with chai.